product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx

```diff
@@ -78,9 +78,9 @@ SELECT bdr.add_commit_scope(
 );
 ```
 
-The first part of the rule states that for any transaction originating on a node in the `origin_node_group` (in this case `location-a`), a majority of the nodes in `location_a` must confirm the transaction before it is committed. 
+The first part of the rule states that for any transaction originating on a node in the `origin_node_group` (in this case `location-a`), a majority of the nodes in `location_a` must confirm the transaction before it's committed. 
```

Comment:
I wasn't sure if passive voice makes sense here. Who does the committing? Is it user action? If so, I would recast to "before you commit it." But I wasn't sure.

Reply:
The confirm is being done by nodes and commit is being done by the system.